health-services/project-factory/src/server/utils/boundaryUtils.ts

```
@@ -66,3 +68,87 @@ export async function getLatLongMapForBoundaryCodes(request:any, boundaryCodeLis
   return boundaryCodeLatLongMap;  
 }
+
+
+export async function getAllBoundariesForCampaign(campaignDetails: any) {
+  const { boundaries } = campaignDetails;
+  const boundaryDataFromRootOnwards = await getBoundaryRelationshipDataFromCampaignDetails(campaignDetails);
+  let allBoundaries = [];
+  var boundaryChildren = boundaries?.reduce((acc: any, boundary: any) => {
+    acc[boundary.code] = boundary?.includeAllChildren;
+    return acc;
+  }, {});
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)